Aspire Budgeting/Views/TabBarView/ProminentTabBarItemView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ProminentTabBarItemView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct ProminentTabBarItemView: View {
+  
+  var width: CGFloat = 70
+  
+  private var innerCircleWidth: CGFloat {
+    return width - 10
+  }
+  
+  private var imageWidth: CGFloat {
+    return innerCircleWidth / 2
+  }
+  
+  private var gradient: LinearGradient {
+    let endColor = Color(red: 0.495, green: 0.945, blue: 0.571)
+    
+    let startColor = Color(red: 0.176, green: 0.784, blue: 0.59)
+    
+    let gradient = Gradient(colors: [startColor, endColor])
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/TabBarView/ProminentTabBarItemView.swift#L26"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ProminentTabBarItemView.swift:26</code></td>
    </tr>
  </tbody>
</table>